test/test_examples_2d_unstructured_quad.jl

```diff
@@ -58,11 +58,11 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
 
   @testset "elixir_ape_unstructured_quad_gauss_wall.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_ape_unstructured_quad_gauss_wall.jl"),
-      l2   = [0.028478721740084537, 0.028283869218625128, 0.06602783645566432, 0.0,
+      l2   = [0.029331247985489625, 0.02934616721732521, 0.03803253571320854, 0.0,
               7.465932985352019e-16, 1.4931865970704038e-15, 1.4931865970704038e-15],
-      linf = [0.264595205647784, 0.2618426663784852, 1.1513244355909877, 0.0,
+      linf = [0.3626825396196784, 0.3684490307932018, 0.8477478712580901, 0.0,
               8.881784197001252e-16, 1.7763568394002505e-15, 1.7763568394002505e-15],
-      tspan = (0.0, 10.0))
+      tspan = (0.0, 5.0))
   end
 end
```

Comment:
Did the Trixi witch ape results change?

Reply:
yes, the solution no longer grows in time now that the bug in the normal flux is fixed. I put a new movie in the Sciebo

Reply:
Nice, thanks!